python/ray/autoscaler/_private/monitor.py

```diff
@@ -203,6 +210,10 @@ def _run(self):
             # round of messages.
             time.sleep(AUTOSCALER_UPDATE_INTERVAL_S)
 
+    def _stop_message_recieved(self):
+        return (self.stop_receiver_connection
+                and self.stop_receiver_connection.poll())
+
```

Comment:
Where is this used?

Reply:
Thanks -- Had originally implemented the stop signaling with a mp.Pipe -- forgot to remove.